Comments/ClientApp/src/components/SubmitResponseDialog/SubmitResponseDialog.js

```diff
@@ -80,105 +81,115 @@ export class SubmitResponseDialog extends PureComponent {
+										className="form__radio"
+										id="respondingAsOrganisation--true"
+										type="radio"
+										name="respondingAsOrganisation"
+										checked={respondingAsOrganisation === "yes"}
+										onChange={fieldsChangeHandler}
+										value={"yes"}
+									/>
+									<label
+										data-qa-sel="respond-yes-responding-as-org"
+										className="form__label form__label--radio"
+										htmlFor="respondingAsOrganisation--true">
+										Yes
+									</label>
+								</div>
 
-						</div>
+								<div className="form__group form__group--radio form__group--inline">
+									<input
+										className="form__radio"
+										id="respondingAsOrganisation--false"
+										type="radio"
+										name="respondingAsOrganisation"
+										checked={respondingAsOrganisation === "no"}
+										onChange={fieldsChangeHandler}
+										value={"no"}
+									/>
+									<label
+										data-qa-sel="respond-no-responding-as-org"
+										className="form__label form__label--radio"
+										htmlFor="respondingAsOrganisation--false">
+										No
+									</label>
+								</div>
 
-						{respondingAsOrganisation === "yes" &&
-						<Fragment>
-							<div className="form__group form__group--text">
-								<label htmlFor="organisationName" className="form__label">
-									<strong>Enter the name of your organisation</strong>
-								</label>
-								<input data-qa-sel="organisation-name" data-hj-whitelist id="organisationName" name="organisationName" value={organisationName}
-									className="form__input" type="text" onChange={fieldsChangeHandler}/>
 							</div>
 
-							{showExpressionOfInterestSubmissionQuestion &&
-								<Fragment>
-									<p>
-										<strong>Would your organisation like to express an interest in formally supporting this quality standard?</strong><br/>
-										<a href="/standards-and-indicators/get-involved/support-a-quality-standard" target="_new">More information</a>
-									</p>
-									<div role="radiogroup" aria-label="Would your organisation like to express an interest in formally supporting this quality standard?">
-										<div className="form__group form__group--radio form__group--inline">
-											<input
-												className="form__radio"
-												id="organisationExpressionOfInterest--true"
-												type="radio"
-												name="organisationExpressionOfInterest"
-												checked={organisationExpressionOfInterest === "yes"}
-												onChange={fieldsChangeHandler}
-												value={"yes"}
-											/>
-											<label
-												data-qa-sel="express-interest-yes"
-												className="form__label form__label--radio"
-												htmlFor="organisationExpressionOfInterest--true">
-												Yes
-											</label>
-										</div>
+							{respondingAsOrganisation === "yes" &&
+							<Fragment>
+								<div className="form__group form__group--text">
+									<label htmlFor="organisationName" className="form__label">
```

Comment:
i think this should be <label for=

rather than htmlFor. htmlFor is just when accessing via property not attribute.

Reply:
https://reactjs.org/docs/dom-elements.html#htmlfor